renku/core/models/provenance/activities.py

```diff
@@ -192,6 +192,11 @@ def paths(self):
         index = set()
 
         for file_ in self.commit.diff(self.commit.parents or NULL_TREE):
+            # ignore deleted files (note they appear as ADDED)
+            # in this backwards diff
+            # TODO: set `deprecatedBy` for deleted paths
+            if file_.change_type == 'A':
```

Comment:
Why do they appear as 'A'? According to [this](https://github.com/gitpython-developers/GitPython/blob/master/git/diff.py#L170) it says it's 'D'.

Reply:
yeah it's confusing - it's because of the way the diff is being generated here. Since the file is deleted in the current commit and the diff direction is going backwards in history it then appears as added.